Project.toml

```diff
@@ -0,0 +1,18 @@
+name = "UnitfulAstro"
+uuid = "f2a1aa4e-5634-4978-a6f6-8c9e368f14ef"
+license = "MIT"
+version = "0.3.0"
```

Comment:
Contrary to whatever I may have told you in the past, this field should be equal to the latest released version, `0.2.0` currently (no strange `-DEV` around).  The version will be bumped in the commit that we'll be used for the new release.